data/notices.json

```diff
@@ -447,6 +447,18 @@
         }
       ],
       "schemaVersion": "1"
+    },
+    {
+      "title": "List stack output change issue.",
```

Comment:
Can you add `(cli)` to the front of this. For example,

```
"title": (cli): List stack output change issue.
```

Reply:
Sure @colifran will get that updated.